README.md

```diff
@@ -9,12 +9,45 @@ Example provider
 
 1. Create a token at https://lichess.org/account/oauth/token/create?scopes[]=engine:read&scopes[]=engine:write
 
-2. Run:
+2. Install your favorite engine. 
+   (Under Windows is it is recommended to place it directly in ```C```. Under Unix choose ```usr/bin/YOUR_ENGINE_FOLDER/program```)
+
+3. Install python3 in case you have not: https://www.python.org/downloads/
+
+4. Install pip3 - Packages 
+
+   First go into your project folder.
+
+   On Windows: 
+
+    ```
+      py -m ensurepip --upgrade
+      py -m pip install -r requirements.txt 
+    ```
+
+   On other systems:
+
+    ```
+      py3 -m ensurepip --upgrade
+      py3 -m pip install -r requirements.txt 
```

Comment:
The correct command should be `python3`. And I guess this is debatable but on Linux, I'd probably install pip via a package manager.